p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
```

Comment:
Ruby actually allows you to have ranges of pretty much anything (well, anything that has a definitive ordered enumerable state).

It means that you can do this:
```
uppercase = "A".."Z"
uppercase.cover? "F" # => true
uppercase.cover? "x" # => false
uppercase.cover? "!" # => false
```

This is mostly informative, because I know the C course spoke a lot about the ASCII values of characters, but it's definitely possible to hide a lot of that in Ruby. So I know this simplifies your `cover?` use below, but it probably makes some of the rest of the code more complicated.